src/gwt/src/org/rstudio/studio/client/workbench/views/output/lint/LintManager.java

```diff
@@ -14,8 +14,10 @@
  */
 package org.rstudio.studio.client.workbench.views.output.lint;
 
+import com.google.gwt.event.shared.HandlerRegistration;
```

Comment:
nit: this import should be bundled with other `com.google.gwt` imports

Reply:
Weird, I don't know why the editor did that! I'll move it.